addons/web/static/src/core/commands/command_palette.scss

```diff
@@ -1,136 +1,38 @@
 .o_command_palette {
-  margin-top: 10vh;
+    margin-top: 10vh;
 
-  &.modal-content {
-    border: none;
-    border-radius: 3px !important;
-    overflow: hidden;
-  }
-
-  > .modal-body {
-    padding: 0;
-    overflow: hidden;
-  }
-
-  &_search {
-    display: flex;
-    align-items: center;
-    justify-content: space-between;
-    background-color: white;
-    border-bottom: 1px solid rgba(0, 0, 0, 0.1);
-    padding: 0.8rem 1em;
-    font-size: 1.5rem;
-
-    .fa-search {
-      color:#C7C7C7;
-    }
-
-    input {
-      border: none;
-      padding: 0;
-      line-height: 1.8em;
-    }
-  }
-
-  &_listbox {
-    cursor: default;
-    max-height: 50vh;
-    overflow-y: auto;
-    position: relative;
-    scrollbar-width: none;
-    -ms-overflow-style: none;
-
-    &::-webkit-scrollbar {
-      display: none;
-    }
-
-    &_empty {
-      padding: 1rem 1.3em;
-      line-height: 1.8em;
-      font-style: italic;
-    }
-
-    .o_command_category {
-      padding: 0.5rem 0;
-    }
-
-    hr {
-      margin: 0;
+    > .modal-body {
+        padding: 0;
+        overflow: hidden;
     }
 
-    .o_command {
-      position: relative;
-      line-height: 1.8em;
-      height: 2.8em;
+    &_listbox {
+        cursor: default;
```

Comment:
`cursor: default;` seems not needed, do you confirm?

Reply:
Without it, if you hover the text/string, we'll have the "text cursor (i beam shape) which isn't great IMO.
But actually I think I'm going to go with a cursor:pointer (on the `o_command` class). It's basically a link to something. 
What do you think @stefanorigano ?